site/content/docs/11-deploying.md

```diff
@@ -61,4 +58,4 @@ When deploying to [Now][], you can pass the environment variable into Now itself
 now -e SAPPER_TIMESTAMP=$(date +%s%3N)
 ```
 
-[Now]: https://zeit.co/now
\ No newline at end of file
+[Now]: https://zeit.co/now
```

Comment:
Shouldn't this line be updated?

Reply:
I will remove this as I directly added a link in line:9